src/packages/documents/documents/property-dataset-context/document-property-dataset-context.token.ts

```diff
@@ -0,0 +1,13 @@
+import { UmbPropertyDatasetContext } from '@umbraco-cms/backoffice/property';
+import { UMB_DOCUMENT_ENTITY_TYPE } from '../entity.js';
+import type { UmbDocumentPropertyDataContext } from './document-property-dataset-context.js';
+import { UmbContextToken } from '@umbraco-cms/backoffice/context-api';
+
+export const IsDocumentVariantContext = (
+	context: UmbPropertyDatasetContext,
+): context is UmbDocumentPropertyDataContext => context.getType() === UMB_DOCUMENT_ENTITY_TYPE;
```

Comment:
regarding the rename to `entityType`. I think this here is a good example where it can be confusing what kind of `type` you get. It could have been some sort of context type.